openshift/osdctl
# osd-111717 Added osd-network-verifier egress test command

Adds "osdctl network verify-egress" subcommand which integrates with osd-network-verifier

Done criteria: 

* osdctl cli tool runs all available (currently egress) tests
* cli shows help and returns output identical to ONV tool

To test locally,

Clone this branch. 
Build osdctl binary by running `make build`. 
Select a subnet and get AWS credentials for it. 

run `AWS_ACCESS_KEY_ID=XXX  AWS_SECRET_ACCESS_KEY=XXXX   AWS_SESSION_TOKEN=XXX  ./dist/<select dist for your os>/osdctl network verify-egress --subnet-id=<subnet-id> --region=<region>`
